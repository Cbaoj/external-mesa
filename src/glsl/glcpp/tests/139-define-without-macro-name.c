#define
Error expected because no macro name is ever given!
