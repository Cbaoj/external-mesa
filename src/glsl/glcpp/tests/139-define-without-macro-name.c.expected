0:1(1): preprocessor error: #define without macro name
Error expected because no macro name is ever given!
